build.sh

```diff
@@ -16,7 +16,7 @@ if ! hash jq 2>/dev/null; then
     exit 1
 fi
 
-if [[ ! -z $(git status -s) ]]; then
+if [[ ! -z $(git status -s | grep -vE 'photon-builder.json') ]]; then
```

Comment:
question: is this a required file or can we put this in a `.gitignore`?

Reply:
It's required to be in the repo I think, but needs modifying for each developer's own environment. Currently those changes have to be committed in order to build the appliance.